platform/node/index.js

```diff
@@ -18,14 +18,21 @@ var Map = function(options) {
 
     return new constructor(Object.assign(options, {
         request: function(req) {
-            request(req, function() {
-                var args = arguments;
-                // Protect ourselves from `request` implementations that try to release Zalgo.
-                // http://blog.izs.me/post/59142742143/designing-apis-for-asynchrony
+            try {
+                request(req, function() {
```

Comment:
Wrapping an async function in a try/catch always set off a🚨  in my head. Which function in here can throw?